src/app/util/model-version.ts

```diff
@@ -10,7 +10,8 @@ export const isMigrateModel = (modelData: any, localVersion: number, modelType:
   } else if (importVersion > localVersion) {
     const isNewMajor = ((Math.floor(importVersion) - Math.floor(localVersion)) >= 1);
     if (isNewMajor) {
-      alert('Cannot load model. Version to load is newer than local. Please close the app and update your local productivity version first, before importing the data.');
+      alert('Cannot load model. Version to load is newer than local. Please close\
```

Comment:
Same thing.

Reply:
Should be ok now, I disabled the max-length for strings